### PR TITLE
implementation: product card images

### PR DESCRIPTION
**Pull request by anerolabs**
The product card should display preview images of the related products. The images which appear on the product card should be the same that appear in the Overview module on the item detail page for that product.

By default, the preview image displayed on each card will be the primary image for that product. This should be the same which first appears on the image detail page’s image gallery.

ticket: https://trello.com/c/KgUfNF3j/29-related-product-card-preview-images

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -182,33 +182,72 @@ h1 {
...
+
+.card-info {
+  display: flex;
+  flex-direction: column;
+  padding: 3px;
+}
+
+.card-category {
+  margin: 0px;
+  text-transform: uppercase;
```
Comment (lbc1013) [inline src/styles.scss:232]: I like how you made them as uppercase using the text-transform! I didn't  even know this exists.

**Review by lbc1013 — APPROVED** [review]
it all looks good to me!

File: src/components/related/ProductCard.jsx [hunk 2]
```diff
@@ -61,7 +62,9 @@ class ProductCard extends React.Component {
       axios(pricePicsRequestConfig)
         .then((response) => {
           const data = response.data.results[0];
-          this.setState({price: data.original_price, salePrice: data.sale_price});
+          this.setState({price: data.original_price,
```
Comment (kjniemela) [inline src/components/related/ProductCard.jsx]: I would add an extra newline before the first item in this object

File: src/components/related/ProductCard.jsx [hunk 3]
```diff
@@ -105,14 +108,20 @@ class ProductCard extends React.Component {
   render() {
     const { productCardId, updateAppProductId } = this.props;
     const { name, category, price, salePrice, rating } = this.state;
+    let primaryImg = this.state.primaryImg;
+
+    if (!primaryImg) { primaryImg = 'https://nayemdevs.com/wp-content/uploads/2020/03/default-product-image.png'; }
```
Comment (kjniemela) [inline src/components/related/ProductCard.jsx]: This might be too long for a one-liner?

File: src/components/related/ProductCard.jsx (hunk 2 above)
Comment (anerolabs) [reply src/components/related/ProductCard.jsx]: That does look cleaner! I'll push the change in a moment

File: src/components/related/ProductCard.jsx (hunk 3 above)
Comment (anerolabs) [reply src/components/related/ProductCard.jsx]: Adjustment made

File: src/styles.scss (hunk 1 above)
Comment (anerolabs) [reply src/styles.scss:232]: :)

**Review by kjniemela — APPROVED** [review]

**Pull request merged**